public/js/init.js

```diff
@@ -16,7 +16,14 @@ const checkAuthenticationStatus = (token) => {
     return localStorage.setItem('card-game-token', '');
   }
   window.user = decoded.user;
-};
 
-checkAuthenticationStatus(token);
+  if (window.location.pathname === "/auth/google/callback"
+    || window.location.pathname === "/auth/facebook/callback"
+    || window.location.pathname === "/auth/twitter/callback"
+    || window.location.pathname === "/auth/github/callback"){
+    window.location = '/';
+  }
+  
+};
 
+checkAuthenticationStatus(token);
```

Comment:
Nice work. However, there should be a new line at the end of this file

Reply:
I have done that..tnx